Documentation/ApiOverview/Fal/UsingFal/ExamplesFileFolder.rst

```diff
@@ -37,11 +37,20 @@ The syntax of argument 1 for getFileObjectFromCombinedIdentifier is
 
    [[storage uid]:]<file identifier>
 
-The storage uid is optional. If it is not specified, the default storage
-(virtual storage with uid=0) is used. In the case of a storage uid=0 the local filesystem is checked
-for the given file. If the file is found, then its local path will be used. If the file is not found,
-then the fileadmin on the public web path will be used.
-The file identifier is adapted accordingly to match the new storage's base path.
+The return value is
+
+.. code-block:: none
+
+   File|ProcessedFile|null
+
+The storage uid is optional. If it is not specified, the default storage 0 will be assumed for the beginning.
```

Comment:
```suggestion
The storage uid is optional. If it is not specified, the default storage 0 will be used.
```

"for the beginning" is rather Denglish. ;-)

Reply:
However this is not true. The storage 0 will not be used in some cases. See the following text.

Reply:
modified into "at the beginning"

Reply:
Then I'd suggest "initially".